packages/components/src/TextField/_docs/TextField.mdx

```diff
@@ -0,0 +1,52 @@
+import { Canvas, Controls, Meta } from "@storybook/blocks"
+import { ResourceLinks, KaioNotification, Installation } from "~storybook/components"
+import { LinkTo } from "~storybook/components/LinkTo"
+import * as TextFieldStories from "./TextField.stories"
+
+<Meta of={TextFieldStories} />
+
+# TextField
+
+<ResourceLinks
+  sourceCode="https://github.com/cultureamp/kaizen-design-system/tree/main/packages/components/src/TextField"
+  figma="https://www.figma.com/file/ZRfnoNUXbGZv4eVWLbF4Az/%EF%B8%8F%F0%9F%96%BC%EF%B8%8F-Component-Gallery?node-id=893%3A155422&t=zlNKMVZd1i6MB4db-1"
+  designGuidelines="https://cultureamp.atlassian.net/wiki/spaces/DesignSystem/pages/3081928705/Text+Field"
+  className="!mb-8"
+/>
+
+<KaioNotification />
+
+<Installation
+  installCommand="yarn add @kaizen/components"
+  importStatement='import { TextField } from "@kaizen/components"'
+/>
+
+## Overview
+
+A text field includes a label and an input field you can type text into.
```

Comment:
and optionally validation and description?

Reply:
```suggestion
A text field includes a label, an input field you can type text into, a description and validation 
```
Ah good point, got these from the design docs 😅